src/tty/unix.rs

```diff
@@ -600,19 +603,21 @@ pub struct PosixTerminal {
     stdin_isatty: bool,
     stdout_isatty: bool,
     pub(crate) color_mode: ColorMode,
+    stream_type: OutputStreamType,
 }
 
 impl Term for PosixTerminal {
     type Mode = Mode;
     type Reader = PosixRawReader;
     type Writer = PosixRenderer;
 
-    fn new(color_mode: ColorMode) -> PosixTerminal {
+    fn new(color_mode: ColorMode, stream_type: OutputStreamType) -> PosixTerminal {
         let term = PosixTerminal {
             unsupported: is_unsupported_term(),
             stdin_isatty: is_a_tty(STDIN_FILENO),
             stdout_isatty: is_a_tty(STDOUT_FILENO),
```

Comment:
@eminence There is another problem here:
`stdout_isatty: is_a_tty(STDOUT_FILENO),`

Reply:
Gah, sorry about this.  Fixed in #156

Reply:
No worry.